src/Native/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -50,9 +50,8 @@ C_FUNC(\Name):
         .cfi_startproc
 .endm
 
-.macro LEAF_END_MARKED Name, Section
```

Comment:
How would the check whether an IP is in one of the helper methods (IsIPInMarkedJitHelper) work without the LEAF_END_MARKED?